src/WindowsAzureMessaging/WindowsAzureMessaging/Helpers/MSLocalStorage.m

```diff
@@ -0,0 +1,66 @@
+//----------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//----------------------------------------------------------------
+
+#import "MSLocalStorage.h"
+#import <Foundation/Foundation.h>
+
+static NSString *const installationKey = @"installation";
```

Comment:
Should these names be namespaced? On the Android side we are using a namespace. These looks like they are just in the standard NSUserDefaults storage.

It would make sense to prefix these with at least "MSNH_LocalStorage_" or something similar.

Reply:
I would like to see if there is a common namespacing approach used by other libraries.

Reply:
Will prepend a common suffix to our user default names